src/VirtualClient/VirtualClient.Actions/Pbzip2/Pbzip2Executor.cs

```diff
@@ -146,7 +146,7 @@ private void CaptureMetrics(IProcessProxy process, string commandArguments, Even
             {
                 bool compression = this.Scenario.Contains("Decompression") ? false : true;
 
-                Pbzip2MetricsParser parser = new Pbzip2MetricsParser(process.StandardOutput.ToString(), compression);
+                Pbzip2MetricsParser parser = new Pbzip2MetricsParser(process.StandardError.ToString(), compression);
```

Comment:
Why is the metric in standard error? What changed?

Reply:
Nothing changes, from starting it was standard error, but as part of other PRs, it was somehow changed to standard output, so changing it back, adding comment as well so that noone change it in future by considering it as a mistake

Reply:
That is so weird but thanks for writing that down

Reply:
Agree it is weird